src/hotspot/cpu/riscv/macroAssembler_riscv.cpp

```diff
@@ -790,10 +795,9 @@ void MacroAssembler::emit_static_call_stub() {
 void MacroAssembler::call_VM_leaf_base(address entry_point,
                                        int number_of_arguments,
                                        Label *retaddr) {
-  int32_t offset = 0;
   push_reg(RegSet::of(t1, xmethod), sp);   // push << t1 & xmethod >> to sp
-  movptr(t1, entry_point, offset, t0);
-  jalr(t1, offset);
+  movptr(t1, RuntimeAddress(entry_point), t0);
+  jalr(t1);
```

Comment:
This movptr + jalr sequence could be further simplified into `rt_call(entry_point)`, which could help save one add instruction.

Reply:
Ah, yes. I think you are right.